server/cmd/mmctl/commands/root.go

```diff
@@ -108,6 +108,11 @@ var RootCmd = &cobra.Command{
 		}
 		quiet := viper.GetBool("quiet")
 		printer.SetQuiet(quiet)
+
+		perPage, err := cmd.Flags().GetInt("per-page")
```

Comment:
Apologies I was away on PTO but I'm back now. Should we check if `err != nil` as well?

Reply:
Not every command defines a `per-page` flag. Hence, an error can be ignroed.

Reply:
I wasn't aware that `GetInt` works that way. Yes, in that case, we should check the error. @thelizardreborn Can you please make the change?